CodeGeneration/Sources/generate-swift-syntax/templates/swiftsyntax/RawSyntaxNodesFile.swift

```diff
@@ -241,12 +241,12 @@ func rawSyntaxNodesFile(nodesStartingWith: [Character]) -> SourceFileSyntax {
 
           for (index, child) in node.children.enumerated() {
             try VariableDeclSyntax("public var \(child.varOrCaseName.backtickedIfNeeded): Raw\(child.buildableType.buildable)") {
-              let exclamationMark = child.isOptional ? nil : TokenSyntax.exclamationMarkToken()
+              let exclamationMark = child.isOptional ? "" : "!"
 
               if child.syntaxNodeKind == .syntax {
-                ExprSyntax("layoutView.children[\(raw: index)]\(exclamationMark)")
+                ExprSyntax("layoutView.children[\(raw: index)]\(raw: exclamationMark)")
```

Comment:
I'm wondering why this change was made. Weren't we planning to phase out the use of `raw:` interpolation?